mlops/parallelm/mlops/mlops_rest_connected.py

```diff
@@ -267,6 +269,51 @@ def get_stat(self, stat_name, ion_id, workflow_node_id, agent_id, pipeline_id, s
                         start=start_time, end=end_time)
         return self._get_url_request_response_as_json(url)
 
+    def post_event(self, pipeline_inst_id, event):
+        """
+        Post event using protobuf format
```

Comment:
is it worth considering a json format for events also? that would help remove events all together?

Reply:
Yes, As discuss with LiorA he want to define the format of all stat/event in a common place using a library to generate code for java/python/scala.